client/landing/gutenboarding/store/types.ts

```diff
@@ -13,6 +14,11 @@ export enum SiteType {
 	STORY = 'story',
 }
 
+export interface SiteVertical {
+	label: string;
+	id: string;
+}
```

Comment:
How about only storing the `id`? Mostly for data normalization purposes; IIUC, the label can always be inferred from `state.verticals`, no?